util/settings/settings.go

```diff
@@ -97,6 +97,8 @@ type OIDCConfig struct {
 	CLIClientID            string                 `json:"cliClientID,omitempty"`
 	RequestedScopes        []string               `json:"requestedScopes,omitempty"`
 	RequestedIDTokenClaims map[string]*oidc.Claim `json:"requestedIDTokenClaims,omitempty"`
+	LogoutURL              string                 `json:"logoutURL,omitempty"`
+	LogoutRedirectURL      string                 `json:"logoutRedirectURL,omitempty"`
```

Comment:
The `LogoutRedirectURL` should be calculated on they fly manually. It is `ArgoCDSettings.URL` + optional `RootPath` from https://github.com/argoproj/argo-cd/blob/69d8831b3886bddacf7b9458cd68676d3fae0dfe/server/server.go#L173